contracts/globalConstraints/GlobalConstraintInterface.sol

```diff
@@ -2,6 +2,14 @@ pragma solidity ^0.4.19;
 
 
 contract GlobalConstraintInterface {
+
+    enum CallPhase { Never, Pre, Post,PreAndPost }
```

Comment:
Why we need `Never`?

Reply:
Well, I still do not really knows :). 
Though I can think on a scenario which one might want to use a dynamic global constraint contract ...which is controlled by some entity which can turn this gc contract to none valid without the need to remove it from the controller....

Though it might be useless and should be removed.

Reply:
What's wrong with adding/removing on demand? Using the `Never` approach requires special logic for permissions etc...

Reply:
Removed at 64c3a9a